keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/config/ClouddriverConfiguration.kt

```diff
@@ -43,6 +43,7 @@ class ClouddriverConfiguration {
       ?: throw BeanCreationException("Invalid URL: $clouddriverBaseUrl")
 
   @Bean
+  @ConditionalOnMissingBean(CloudDriverService::class)
```

Comment:
This is so we can replace the default implementation with the Netflix internal one.